tox.cover.sh

```diff
@@ -41,6 +41,8 @@ cover () {
     min=99
   elif [ "$1" = "certbot_dns_route53" ]; then
     min=92
+  elif [ "$1" = "certbot_dns_yandex" ]; then
+    min=92
```

Comment:
It seems like we should be able to do better here; most DNS plugins can cover all but 1 line without much difficulty. If we don't end up removing lines 77-81, let's make sure we adjust a test to cover them.